Assets/MixedRealityToolkit/_Core/Utilities/ClipPlane.cs

```diff
@@ -12,24 +13,37 @@ namespace Microsoft.MixedReality.Toolkit.Internal.Utilities
     [ExecuteInEditMode]
     public class ClipPlane : MonoBehaviour
     {
+        /// <summary>
+        /// The renderer(s) that should be affected by the clip plane.
```

Comment:
Don't need a summary tag on the private field. Just a public property if one exists.

Reply:
Opps, I'll fix this.